.travis.yml

```diff
@@ -8,6 +8,7 @@ cache:
   directories:
     - ~/.cache
 install:
+  - npm i
```

Comment:
FYI: I had to add this because the initial build failed due to the spinner package